.github/workflows/build_and_release.yml

```diff
@@ -127,6 +127,12 @@ jobs:
             EXTRA_CMAKE_ARGS="-DUSE_BOOST_REGEX=1"
           fi
 
+          if [[ "${ID}" == "ubuntu" && "${VERSION_ID}" == "18.04" ]]; then
+            # Ubuntu 18.04 has system Aubio 0.4.5, this is not enough
+            # because performous requires a minimum version of 0.4.9.
+            EXTRA_CMAKE_ARGS="${EXTRA_CMAKE_ARGS} -DSELF_BUILT_AUBIO=FORCE"
```

Comment:
I guess this can be detected within the FindAubio cmake module right? The error message even gives you the version number it did find.

Doing a simple if there and then fall back to self-built seems appropiate

Reply:
I think I prefer not using AUTO as much as possible. If a specific target requires the "ALWAYS" target put it, else we should leave it to NEVER.
Other "AUTO" are currently only set until the dependency is added to the docker image (or dependency download)